### PR TITLE
refactor(kafka): Set JMX exporter port configurable with default 18202

### LINKED ISSUE #79: Change the range of the JMX ports to 182XX
To avoid issues like https://github.com/TOSIT-IO/tdp-collection/pull/535, we should change the range of the JMX ports to 182XX

### PR DESCRIPTION
**Pull request by dapicard**
#### Which issue(s) this PR fixes

Fixes #79 

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: tdp_vars_defaults/kafka/kafka.yml [hunk 1]
```diff
@@ -157,7 +157,8 @@ jmxremote_password: Tdpjmx123,
 jmx_exporter_root_dir: /opt/tdp/jmx-exporter
 jmx_exporter_install_file: "{{ jmx_exporter_root_dir }}/jmx-exporter.jar"
 jmx_common_opts: "-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.authenticate=true -Dcom.sun.management.jmxremote.ssl=true"
-jmx_exporter_kafka_opts: "{% if 'exporter_jmx' in groups and groups['exporter_jmx'] %}-javaagent:{{ jmx_exporter_install_file }}=18120:{{ jmx_exporter_conf_dir }}/kafka.yml{% else %}{% endif %}"
+jmx_exporter_kafka_port: 18202
```
Comment (rpignolet) [inline tdp_vars_defaults/kafka/kafka.yml:160]: Can you use the same name pattern and move it in `tdp-cluster` file ?

You can see a name pattern example here: https://github.com/TOSIT-IO/tdp-collection/blob/8c1a55ba6b8dbad10646148e16905c659eccb50b/tdp_vars_defaults/tdp-cluster/tdp-cluster.yml#L94

You can see the usage of this variable here: https://github.com/TOSIT-IO/tdp-collection/blob/8c1a55ba6b8dbad10646148e16905c659eccb50b/tdp_vars_defaults/hadoop/hadoop.yml#L47

Comment (dapicard) [reply tdp_vars_defaults/kafka/kafka.yml:160]: There is no tdp_vars_default/tdp-cluster/tdp-cluster.yml file in this repository. Should I create it ?

Comment (rpignolet) [reply tdp_vars_defaults/kafka/kafka.yml:160]: Yes you can create it.

Comment (dapicard) [reply tdp_vars_defaults/kafka/kafka.yml]: Done.

**Review by rpignolet — APPROVED** [review]
Nice work !

**Review by leopaul36 — APPROVED** [review]

**Pull request merged**